Streetcode/Streetcode.BLL/DTO/Partners/CreatePartnerDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Streetcode.BLL.DTO.AdditionalContent;
+using Streetcode.BLL.DTO.Partners;
+using Streetcode.BLL.DTO.Streetcode;
+
+namespace Streetcode.BLL.DTO.Partners
+{
+    public class CreatePartnerDTO
+    {
+        public int Id { get; set; }
+        public bool IsKeyPartner { get; set; }
+        public string Title { get; set; }
+        public string? Description { get; set; }
+        public string TargetUrl { get; set; }
+        public int LogoId { get; set; }
+        public string? UrlTitle { get; set; }
+        public string LogoBase64 { get; set; }
```

Comment:
The field is named as if the base64 format is stored here, but isn't it? Is there only a name?

Reply:
this is nowhere used now
it was created because we thought the image would be sent with the entity, but as you know, it is not, so it needs to be removed. Thanks)